apiserver/facades/controller/raftlease/register.go

```diff
@@ -13,23 +13,11 @@ import (
 
 // Register is called to expose a package of facades onto a given registry.
 func Register(registry facade.FacadeRegistry) {
-	registry.MustRegister("RaftLease", 1, func(ctx facade.Context) (facade.Facade, error) {
```

Comment:
It looks like this is being snuck in with no explanation, can we have a separate commit with info please?

Keep going to remove V1 from the facade.go file too.